### DISCUSSION TITLE
Option environment TRIVY_CACHE_DIR

### DISCUSSION THREAD

**Discussion by bryanhermass** [aquasecurity/trivy#7752] [Q&A]
### Question

Hi guys!

I have a problem with too many requests to GET https://ghcr.io/v2/aquasecurity/trivy-db/manifests/2: TOOMANYREQUESTS: retry-after: 958.651µs, allowed: 44000/minute.

I found an option to create a local cache by setting the environment variable TRIVY_CACHE_DIR=<path> and using the command trivy --download-db-only. It downloads the database to the specified path, and when scanning images, it won't connect to https://ghcr.io/v2/aquasecurity/trivy-db/manifests/2.

I think this could solve the issue. Is this a viable option or not? My concern is whether the source from which this database is downloaded is always up to date. Can I trust this option?

### Target

Container Image

### Scanner

Misconfiguration

### Output Format

None

### Mode

Standalone

### Version

```bash
Version: 0.18.3
```

**Comment by DmitriyLewen** [comment]
Discussion about `TOOMANYREQUESTS` - https://github.com/aquasecurity/trivy/discussions/7699
